eli5/lightning.py

```diff
@@ -16,7 +16,6 @@
 from eli5.explain import explain_prediction, explain_weights
 
 
-@explain_weights.register(BaseEstimator)
```

Comment:
Does it mean that e.g. for SVRGClassifier `explain_weight_lighting` won't be called, and eli5.explain_weights will be called instead?